components/boards/espressif_saola_1_wrover/board.h

```diff
@@ -36,7 +36,7 @@
 
 // GPIO that implement 1-bit memory with RC components which hold the
 // pin value long enough for double reset detection.
-// #define PIN_DOUBLE_RESET_RC
+#define PIN_DOUBLE_RESET_RC 13
```

Comment:
saola doesn't have this feature implementation, I think we should leave this as local modification for testing.